test/integration/controller/job/job_controller_test.go

```diff
@@ -845,7 +845,7 @@ var _ = ginkgo.Describe("Job controller interacting with scheduler", ginkgo.Orde
 		util.ExpectAdmittedActiveWorkloadsMetric(prodClusterQ, 1)
 	})
 
-	ginkgo.It("Should unsuspend job iff localQueue is in the same namespace", func() {
+	ginkgo.It("Should unsuspend job if localQueue is in the same namespace", func() {
```

Comment:
This was correct https://en.wikipedia.org/wiki/Iff

Reply:
Reverted in a5bfc265